packages/web/src/pages/app/contributions.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { NextPage } from 'next';
+import { Heading, SimpleGrid, Text, VStack } from '@chakra-ui/react';
+import { AppLayout } from '../../components/Layout';
+import { ContributionsBox } from '../../components/Contributions';
+
+export interface ContributionsList {
```

Comment:
We should keep this variable name close to the actual structure that it represents. 🙂 
```suggestion
export interface Contribution {
```